sdk/search/search-documents/src/searchIndexingBufferedSenderImpl.ts

```diff
@@ -91,10 +91,11 @@ class SearchIndexingBufferedSenderImpl<T> implements SearchIndexingBufferedSende
   /**
    * Creates a new instance of SearchIndexingBufferedSender.
    *
-   * @param client Search Client used to call the underlying IndexBatch operations.
-   * @param options Options to modify auto flush.
+   * @param client - Search Client used to call the underlying IndexBatch operations.
+   * @param options - Options to modify auto flush.
    *
    */
+  // eslint-disable-next-line @azure/azure-sdk/ts-use-interface-parameters
```

Comment:
My original plan was to create a Interface SearchClient and Change the current SearchClient to SearchClientImpl. But after creating it, I realized it did not make much sense. I have to expose both the Interface and Implementation to the user (unlike BufferSenderImpl which is not exposed). So, I do not see much use for the Interface. So, disabled the error here.

Reply:
Since this class only uses a single method on `SearchClient`, why don't we make a small interface that describes this?

```ts
interface IndexDocumentsClient<T> {
 indexDocuments(
    batch: IndexDocumentsBatch<T>,
    options: IndexDocumentsOptions
  ): Promise<IndexDocumentsResult>
}
```

Since `SearchClient` satisfies the above, it will work without having to modify `SearchClient` at all.

Reply:
Done